documentation/status-logging.md

```diff
@@ -0,0 +1,54 @@
+# Status logging
+
+TK-status can periodically log status data to a file in JSON format.
+
+To take advantage of this you must do two things:
+
+1. Enable logging though you TK app's configuration. For example, to log every 2 seconds, your configuration might look like this:
+
+    ```
+    status: {
+        logging-frequency: 2000,
+        status-logging-enabled: true
+    }
```

Comment:
Those names sound good to me

I kind like your layout @rlinehan regardless of if the settings might expand. Seems cleaner to separate it out like that